locales/en/bento.ftl

```diff
@@ -0,0 +1,10 @@
+bento-button-title = Firefox apps and services
+fx-makes-tech = Firefox is tech that fights for your online privacy.
+made-by-Mozilla = Made by Mozilla
+fx-desktop = Firefox Browser for Desktop
+fx-mobile = Firefox Browser for Mobile
```

Comment:
@flodolo  Do you know if there is a consensus on how these should be localized? I'm assuming the "Browser for Desktop" and "Browser for Mobile" bits should be translated.

Reply:
I kind of remember seeing a conversation about considering the whole "Firefox Browser" as a brand, but I'm not sure who can confirm if there are guidelines. I wouldn't be super happy if that's the case, to be honest.

Reply:
Given the conversation that happened in Slack, I think we need to go back to your previous solution, with `Firefox Browser` defined as a term (and a clear comment that it needs to stays in English), and only that referenced in these strings.

It's unfortunate, but that's the only way to enforce that the term stays untranslated :-\